test/resources/Entity.tests.ts

```diff
@@ -658,15 +658,15 @@ describe('Entities - core methods tests', () => {
           id: entities_create_connect_subscription_response.id,
           name: 'connect',
           status: 'active',
-          latest_request_id: entities_create_connect_subscription_response.latest_request_id,
```

Comment:
minor thing but if `entities_create_connect_subscription_response.latest_request_id` returns `null` anyways then why hardcode the `null` value in? i added this last time to stay consistent with the way we have `expect_results.connect.id: entities_create_connect_subscription_response.id`.